patterns/hero-overlapped-book-cover-with-links.php

```diff
@@ -99,9 +99,7 @@
 							<!-- /wp:spacer -->
 
 							<!-- wp:paragraph {"fontSize":"medium"} -->
-							<p class="has-medium-font-size">
-								<?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>.', 'Hero - Overlapped book cover pattern bottom text', 'twentytwentyfive' ) ); ?>
-							</p>
+							<p class="has-medium-font-size"><?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>', 'Sample text with link.', 'twentytwentyfive' ) ); ?></p>
```

Comment:
```suggestion
							<p class="has-medium-font-size"><?php echo wp_kses_post( _x( 'Outside Europe? View <a href="#" rel="nofollow">international editions</a>.', 'Sample text with link.', 'twentytwentyfive' ) ); ?></p>
```

Re-add the missing period.